docs/configuration.md

```diff
@@ -97,6 +97,18 @@ storage:
 		poolname: radospool
 		username: radosuser
 		chunksize: 4194304
+	swift:
+		username: username
+		password: password
+		authurl: https://storage.myprovider.com/v2.0 or https://storage.myprovider.com/v3/auth
```

Comment:
Suggest that we add an example for v1.0 in authurl too. So the line can be like:
authurl: https://storage.myprovider.com/auth/v1.0 or https://storage.myprovider.com/v2.0 or https://storage.myprovider.com/v3/auth